docs/contribute.rst

```diff
@@ -64,13 +64,34 @@ sure that you have all required packages installed on your box::
 
     sudo dnf install gcc {python3,libvirt,krb5,libpq}-devel
 
+You may also need following packages if they are not already
+installed on your system::
+
+    sudo dnf install git python3-docutils
```

Comment:
Hm, How about moving `python3-docutils` to the line with gcc and -devel?

If we want to be user friendly then one also needs `make`

Reply:
That's true :) I am fixing that right now.

Reply:
Done. There was also one specific package only necessary for CentOS Stream 8 I kept in separate line.